modules/mbedtls/configs/config-tls-generic.h

```diff
@@ -482,11 +482,13 @@
 #define MBEDTLS_PSA_P256M_DRIVER_ENABLED
 #endif
 
-#if defined(CONFIG_ARCH_POSIX)
+#if defined(CONFIG_ARCH_POSIX) && !defined(CONFIG_PICOLIBC)
 #define MBEDTLS_PSA_KEY_SLOT_COUNT 64 /* for BLE Mesh tests */
+#define MBEDTLS_PSA_ITS_FILE_C
+#define MBEDTLS_FS_IO
 #endif
 
-#if defined(CONFIG_SECURE_STORAGE)
+#if defined(CONFIG_SECURE_STORAGE) || (defined(CONFIG_ARCH_POSIX) && !defined(CONFIG_PICOLIBC))
```

Comment:
See the discussion in https://github.com/zephyrproject-rtos/zephyr/issues/79533.
I also don't like this, but it was this way before the introduction of the secure storage subsystem.
This could be improved in a follow-up PR. Here the goal is to fix CI.

Reply:
Since CI passed now and we need this hotfix, perhaps this can be solved later?